f2/src/components/formik/alert/index.js

```diff
@@ -70,7 +77,15 @@ export const ErrorSummary = (props) => {
                 return (
                   <React.Fragment key={key}>
                     <Row>
-                      <A href={`#${key}`} marginBottom="0.5rem" color="initial">
+                      <A
+                        href={`#${key}`}
```

Comment:
This can be written as          
href="javascript:void(0)"

Reply:
This causes an error in the console (dev deployment). An issue was created to remove this type of property previously - #1990 . The `preventDefault()` will prevent navigation via href.